Makefile

```diff
@@ -9,11 +9,11 @@ clean:
 
 deps: node_modules
 
-node_modules: package.json yarn.lock
+node_modules: yarn.lock
 	@echo "install frontend dependencies"
 	yarn install --pure-lockfile --no-progress
 
-build:
+build: clean
```

Comment:
Why is it necessary to clean before building? Just asking since I don't know :)

Reply:
Good question. Couldn't build if the build directory existed, but now looking at it think "phony" maybe are needed to not confuse make with command and directories/files?

Reply:
Added phony and cleaned up the makefile.